src/gl/directx11/vcRenderShaders.cpp

```diff
@@ -971,8 +980,28 @@ const char* const g_FlatColour_FragmentShader = R"shader(
   }
 )shader";
 
+const char *const g_DepthOnly_FragmentShader = R"shader(
+  struct PS_INPUT
+  {
+    float4 pos : SV_POSITION;
+  };
+
+  struct PS_OUTPUT
+  {
+    float4 Color0 : SV_Target;
+  };
```

Comment:
This shader is used with depth writing enabled, so the hardware will write out a depth for us. Explicitly writing SV_Depth here (while it can work) will disable any depth testing optimizations (depth testing happens before fragment shader pass).